docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,827 @@
+
+<div class="aligncenter">
+  {{<imgproc src="/tutorials/air-quality-fleet/three-sensor-dash.png" resize="x900" declaredimensions=true alt="Air quality dashboard with PM2.5 readings from three different sensor machines displayed." style="max-width:400px" >}}
+</div>
+
+## Requirements
+
+You can complete this tutorial using any number of air quality sensing machines.
+Your {{< glossary_tooltip term_id="machine" text="machines" >}} can be in different {{< glossary_tooltip term_id="location" text="locations" >}} but you should keep them all in one {{< glossary_tooltip term_id="organization" text="organization" >}} for simplicity and so that you can follow along with this tutorial more easily.
+Viam's fleet management system allows you to pull data from any machines you can authenticate to; if you'd like to pull data from multiple organizations that is possible, but is not covered within this tutorial.
+
+For each machine, you will need the following:
+
+- [SDS011 Nova PM sensor](https://www.amazon.com/SDS011-Quality-Detection-Conditioning-Monitor/dp/B07FSDMRR5)
+  - If you choose to use a different air quality sensor, you may need to [create your own module](/registry/create/) implementing the [sensor API](/components/sensor/#api) for your specific hardware.
+- A single-board computer (SBC) with [`viam-server` installed](https://docs.viam.com/get-started/installation/) and connected to the [Viam app](https://app.viam.com)
+- An appropriate power supply
+
+In addition to `viam-server`, this tutorial uses the following software:
+
+- [`sds0011` sensor module](https://github.com/zaporter/viam-sds011)
+- The [Viam TypeScript SDK](https://ts.viam.dev/)
+
+## Set up your hardware
+
+1. For each sensing machine, connect the PM sensor to a USB port on the machine's SBC.
+2. Enable serial communication on each SBC.
+   For example, if you are using a Raspberry Pi, SSH to it and [enable serial communication in `raspi-config`](/get-started/installation/prepare/rpi-setup/#enable-communication-protocols).
+
+3. Position your sensing machines in strategic locations, and connect them to power.
+   Here are some ideas for where to place sensing machines:
+
+   - At home:
+     - In an outdoor location protected from weather, such as under the eaves of your home
+     - In the kitchen, where cooking can produce pollutants
+     - Anywhere you spend lots of time indoors and want to measure exposure to pollutants
+   - At work:
+     - At your desk to check your exposure throughout the day
+     - Near a door or window to see whether pollutants are leaking in
+
+## Configure your air quality sensors
+
+You need to [configure](/build/configure/) your hardware so that each of your machines can communicate with its attached air quality [sensor](/components/sensor/).
+
+No matter how many sensing machines you use, you can configure them all very quickly using [fragments](/fleet/configure-a-fleet/).
```

Comment:
Briefly introduce fragments - assume that people don't know what those are, so they'll be confused at this point. Explain how you wouldn't want to manually set up the same configuration many times and therefor you'll use a reusable configuration called a Fragment. possibly with a glossary item on it instead of the link?